huxley/www/js/components/RegistrationView.js

```diff
@@ -112,301 +116,187 @@ var RegistrationView = React.createClass({
 
   render: function() {
     var conference = this.context.conference;
+
+    //I feel like this (And a lot of others in this section) could 
+    //Be reduced with some kind of mapping,
+    //But there's no obvious way to map over Objects in JS so :/
+    var accountInformationErrors = {
```

Comment:
For now at least I would leave them out and stick to the conventions we've been using; later on we can work out if we prefer a different structure